packages/nvidia_nat_eval/src/nat/plugins/eval/exporters/file_eval_callback.py

```
@@ -0,0 +1,160 @@
+        if output_dir is None:
+            logger.debug("FileEvalCallback: no output_dir on EvalResult, skipping file export")
+            return
+
+        output_dir.mkdir(parents=True, exist_ok=True)
+
+        self._write_configuration(result, output_dir)
+        self._write_workflow_output(result, output_dir)
+        self._write_evaluator_outputs(result, output_dir)
+
+    # ------------------------------------------------------------------
+    # Private helpers – logic extracted from EvaluationRun
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)